drivers/network/tcpip/lwip/doc/doxygen/generate.bat

```diff
@@ -0,0 +1 @@
+doxygen lwip.Doxyfile
```

Comment:
Can we strip that doxgen-part when importing the updated library?
![image](https://github.com/user-attachments/assets/c67fcf1b-2e76-459d-ae47-f5fafdac64e0)

Reply:
My goal is to keep as intact as possible lwIP sources.
Doxygen is an optional dependency, so I don't see a problem to have 1 more line printed during configure step.

Reply:
I want at least others to see that discussion-thread, unresolving it for that sole purpose.